packages/runtime/container-runtime/src/runWhileConnectedCoordinator.ts

```diff
@@ -59,38 +89,21 @@ export class RunWhileConnectedCoordinator {
      * The promise will also resolve if stopped either externally or by disconnect.
      * The return value indicates whether the start is successful or not.
```

Comment:
Remove this part of the comment as this function doesn't return anything anymore.